src/components/style/Calculator.css

```diff
@@ -0,0 +1,54 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  display: grid;
+} */
```

Comment:
- [ ] It's a good practice not to have commented unused code in our codebase. Kindly remove this unused commented code so your codebase will look clean and professional. 👍🏽

Reply:
@AmaduKamara, thank you for the review and good explanation appreciate it! 🥇